example/eblob_backend.cpp

```diff
@@ -85,22 +101,19 @@ int blob_read_and_check_stamp(const eblob_backend_config *c,
                               int fd,
                               uint64_t data_offset,
                               uint64_t data_size) {
-	std::array<char, sizeof(eblob_disk_control) + sizeof(dnet_ext_list_hdr)> stamp;
-
-	if (data_size < stamp.size()) {
+	if (timestamp && timestamp->tsec > DNET_SERVER_SEND_BUGFIX_TIMESTAMP) {
```

Comment:
Why was it changed from `!timestamp || timestamp->tsec > DNET_SERVER_SEND_BUGFIX_TIMESTAMP`?

Reply:
No timestamp = we cannot decide to prevent check - isn't it?

But I see that `blob_read_and_check_stamp` never called with `timestamp == nullptr`, so we can remove any check for empty ts.

Reply:
I reverted that change.